.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: go
 
 go:
-  - 1.8
```

Comment:
The latest version of the linter does not support Go 1.8 (it will not compile). Given that the current version of Go is 1.11.x, I think that just testing for Go 1.9, Go 1.10 and tip should be sufficient (major version - 2)